src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -307,6 +324,71 @@ private BoundExpression CreateMethodGroupConversion(CSharpSyntaxNode syntax, Bou
             return new BoundConversion(syntax, group, conversion, @checked: false, explicitCastInCode: isCast, constantValueOpt: ConstantValue.NotAvailable, type: destination, hasErrors: hasErrors) { WasCompilerGenerated = source.WasCompilerGenerated };
         }
 
+        private BoundExpression CreateTupleConversion(CSharpSyntaxNode syntax, BoundExpression source, Conversion conversion, bool isCast, TypeSymbol destination, DiagnosticBag diagnostics)
```

Comment:
It seems like the overal style used in the caller CreateConversion is to pass unconverted source, so I did not want this one to stand out.